taichi/backends/cuda/codegen_cuda.cpp

```diff
@@ -105,41 +105,40 @@ class CodeGenLLVMCUDA : public CodeGenLLVM {
   }
 
   void visit(PrintStmt *stmt) override {
+    // TODO(@archibate): ask @yuanming-hu about how to deal with CUDA print
+
     TI_ASSERT(stmt->width() == 1);
 
-    auto value_type = tlctx->get_data_type(stmt->stmt->ret_type.data_type);
+    std::vector<llvm::Type *> types;
+    std::vector<llvm::Value *> values;
 
-    std::string format;
+    std::string formats;
+    for (auto &&content : stmt->contents) {
```

Comment:
Here and below: Why `&&`? Note that `&&` usually conveys special meanings in C++ (e.g. move), which isn't the case here. Can we use `auto`, given that `x` is just a raw pointer and is cheap to copy?